vllm/v1/core/scheduler.py

```diff
@@ -324,23 +330,24 @@ def schedule(self) -> "SchedulerOutput":
         # Construct the scheduler output.
         new_reqs_data = [
             NewRequestData.from_request(req,
-                                        req_to_new_block_ids[req.request_id],
-                                        req.num_computed_tokens)
+                                        req_to_new_block_ids[req.request_id])
             for req in scheduled_new_reqs
         ]
         resumed_reqs_data = [
             self._make_cached_request_data(
                 req,
+                num_scheduled_tokens[req.request_id],
+                scheduled_spec_decode_tokens.get(req.request_id, []),
```

Comment:
nit: it's a bit cheaper to use `()` instead of `[]` for this kind of thing (but there's probably a bunch of other places we coudl change too)

Reply:
Oh didn't know that! Fixed!